app/api/tickets.py

```diff
@@ -182,6 +182,11 @@ def before_update_object(self, ticket, data, view_kwargs):
                 raise UnprocessableEntity(
                     {'event_id': ticket.event.id}, "Event having paid ticket must have a payment method")
 
+        if ticket.deleted_at != data.get('deleted_at'):
+            if not has_access('is_coorganizer', event_id=ticket.event_id):
+                raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
```

Comment:
This is already checked below in decorators

Reply:
Ok will remove